bundles/admin/admin-layereditor/view/LocalizingFlyout.jsx

```diff
@@ -9,7 +9,7 @@ const ExtraFlyout = Oskari.clazz.get('Oskari.userinterface.extension.ExtraFlyout
 
 // NOTE: Everything is in the single file for demonstrating purposes.
 export class LocalizingFlyout extends ExtraFlyout {
-    constructor (instance, title, options = {}, deleteMapLayersText) {
+    constructor(instance, title, options = {}, deleteMapLayersText) {
```

Comment:
This file has ONLY formatting changes? No added/modified/removed code?

Reply:
Something happened with a developer's IDE that formatted everything weirdly but I think I got to revert them all in the lastest push.